fsw-architecture/operating-mode-as-fsm.puml

```diff
@@ -22,14 +22,17 @@ state FiniteStateMachine {
       state MissionMode1
             MissionMode1 : Record impedence probe data to event log
             MissionMode1 : for next 2 minutes at 1 second intervals
+            MissionMode1 : Record TLE entering and existing MM1
```

Comment:
I would split this into lines 23 and 25, one for entering and one for exiting mission mode